web/src/app/services/loi/loi.service.ts

```diff
@@ -78,26 +79,17 @@ export class LocationOfInterestService {
     return this.lois$;
   }
 
-  static getAnonymousDisplayName({
-    loi,
-    index,
-  }: {
-    loi: LocationOfInterest;
-    index: number;
-  }): string {
-    const geometryType = loi.geometry?.geometryType
-      .toLocaleLowerCase()
-      .replace(/_/g, ' ');
-    return `Unnamed ${geometryType} ${index + 1}`;
+  static getAnonymousDisplayName(loi: LocationOfInterest): string {
+    const geometryType = loi.geometry?.geometryType;
+    return `Unnamed ${geometryType === GeometryType.POINT ? 'point' : 'area'}`;
```

Comment:
And let's just define two strings, "Unnamed point" and "Unnamed area", instead of using a string template, to make it easier to translate later.